packages/react/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@axiomhq/react",
+  "version": "0.0.1",
+  "description": "",
+  "type": "module",
+  "main": "dist/cjs/index.cjs",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "exports": {
+    "./transports": {
+      "import": {
+        "types": "./dist/esm/transports/index.d.ts",
+        "default": "./dist/esm/transports/index.js"
+      },
+      "require": {
+        "types": "./dist/cjs/transports/index.d.cts",
+        "default": "./dist/cjs/transports/index.cjs"
+      }
+    },
+    ".": {
+      "import": {
+        "types": "./dist/esm/index.d.ts",
+        "default": "./dist/esm/index.js"
+      },
+      "require": {
+        "types": "./dist/cjs/index.d.cts",
+        "default": "./dist/cjs/index.cjs"
+      }
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "vite build"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@axiomhq/logging": "workspace:*",
+    "@types/react": "^19",
```

Comment:
do the types need to in deps or should we move this to "devDeps"?

Reply:
Moved this to devDependencies